client-registry-jempi/docker-compose.api.yml

```diff
@@ -24,6 +24,12 @@ services:
       - "jempi-shared-data:/app/csv"
     deploy:
       replicas: ${JEMPI_API_INSTANCES}
+      labels:
+        - traefik.enable=true
+        - traefik.docker.network=reverse-proxy-traefik_public
+        - traefik.http.routers.jempi-api.service=jempi-api
+        - traefik.http.services.jempi-api.loadbalancer.server.port=50000
+        - traefik.http.routers.jempi-api.rule=Host(`${JEMPI_API_TRAEFIK_SUBDOMAIN}.${DOMAIN_NAME_HOST_TRAEFIK}`)
```

Comment:
Do we need a subdomain for Jempi API or it can run under the Jempi web subdirectory?

Reply:
Yes we need a subdomain, sub-directories do not work for any of the Jempi services